core/controllers/translator.py

```diff
@@ -179,6 +179,7 @@ def post(self, exploration_id):
 
 
 class ExplorationTranslationHandler(base.BaseHandler):
+    """Handles given exploration."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
```

Comment:
Docstring in line 188 needs args, raises, etc.

Reply:
Done